verticapy/tests_new/performance/vertica/test_qprof_simple.py

```diff
@@ -0,0 +1,100 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import pytest
+import logging
+
+import verticapy as vp
+import vertica_python
+
+from verticapy.performance.vertica import QueryProfiler
+from verticapy.datasets import load_amazon
+from verticapy.core.vdataframe import vDataFrame
+from io import StringIO
+
+
+def test_profile_simple():
+    """Create a query profiler and run the steps on a simple query"""
+    assert len(vp.available_connections()) > 0
+    amzn = setup_dummy_table_run_query()
+    request = f"""
+    SELECT 
+        date, 
+        MONTH(date) AS month, 
+        AVG(number) AS avg_number 
+    FROM 
+        public.amazon 
+    GROUP BY 1;
+    """
+    qp = QueryProfiler(request)
+    check_version(qp)
+    check_request(qp, "avg(number) AS avg_number")
+
+
+def setup_dummy_table_run_query() -> vDataFrame:
```

Comment:
we have data load fixture define in conftest file present at VerticaPy/verticapy/tests_new/conftest.py . Please use it. if you want to define any new data and can be shared by other test. Please defined under this conftest file.

Reply:
schema_loader added with commit 42001a92

Reply:
setup_dummy_table_run_query function is not needed. You can directly use amazon_vd fixture defined in VerticaPy/verticapy/tests_new/conftest.py.